packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -364,7 +376,7 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
         const correction = isRtl
           ? tabsMeta.scrollLeftNormalized + tabsMeta.clientWidth - tabsMeta.scrollWidth
           : tabsMeta.scrollLeft;
-        startValue = tabMeta.left - tabsMeta.left + correction;
+        startValue = (isRtl ? -1 : 1) * (tabMeta[start] - tabsMeta[start] + correction);
```

Comment:
This looks wrong, see https://github.com/mui-org/material-ui/blob/8cc7dd0568adde2b8c23804fabdadbd7cfff21d6/packages/material-ui-utils/src/scrollLeft.js#L7

Reply:
From what I tested with
- Chrome 91 ✅
- Edge 91 ✅
- Firefox 88 ✅
- Safari 14.0.3 ✅
The calculation is correct, it just needs to be negative for `rtl`